text_recognizer/metadata/iam.py

```diff
@@ -6,26 +6,4 @@
 EXTRACTED_DATASET_DIRNAME = DL_DATA_DIRNAME / "iamdb"
 
 DOWNSAMPLE_FACTOR = 2  # If images were downsampled, the regions must also be.
-LINE_REGION_PADDING_X = 8  # add this many pixels around the exact coordinates
-LINE_REGION_PADDING_Y = 8  # add this many pixels around the exact coordinates
-WORD_REGION_PADDING_X = 4
-WORD_REGION_PADDING_Y = 2
-
-PUNCTUATIONS = [
-    "!",
-    '"',
-    "#",
-    "&",
-    "'",
-    "(",
-    ")",
-    "*",
-    "+",
-    ",",
-    "-",
-    ".",
-    "/",
-    ":",
-    ";",
-    "?",
-]
+LINE_REGION_PADDING = 16  # add this many pixels around the exact coordinates
```

Comment:
16 is too much vertically (along the y-axis).

Reply:
infact, reducing this + fixing the image augmentation (random scaling limited to <=1) is what lead to the improved performance in https://wandb.ai/fullstackdeeplearning/fsdl-text-recognizer-2022-training/runs/pdos3fm2

Reply:
> 16 is too much vertically (along the y-axis).

I disagree -- I commented about this on [a now-closed PR](https://github.com/full-stack-deep-learning/fsdl-text-recognizer-2022/pull/31#discussion_r941966326), but the upshot is that I think the extra "letter bits" are a nice form of data augmentation.

reduced performance imo means we are accurately detecting that the models are not learning to avoid these kinds of nuisances

Reply:
here's an example image where i quite like what the generous padding has added:

![848b6021ba630e26e6ecb4616a078d72](https://user-images.githubusercontent.com/10442975/184014719-b180d86d-1dcd-4419-be74-0fefa1779f62.png)

distracting bits of letters added in between lines, a line of ink, even the cutoff line at the bottom is realistic -- users would want to be able to upload images with text cut off by a crop and not confuse the model